.config/.eslintrc

```diff
@@ -15,6 +15,7 @@
       "plugins": ["deprecation"],
       "files": ["src/**/*.{ts,tsx}"],
       "rules": {
+        "no-redeclare": "warn",
```

Comment:
Is there any particular reason you are changing this to a warning?

Reply:
yes, when i changed the types as you requested
> these two types are almost the same. extract the common part. e.g; `QueryCalculation`

I used const/type instead of enum, but I got an error for defining it twice. That's why I changed it to be a warning:
clock-panel/src/types.ts
40:13  warning  'CountdownQueryCalculation' is already defined  no-redeclare
46:13  warning  'CountupQueryCalculation' is already defined    no-redeclare

I agree that I shouldn't have disabled it overall. Now I disable it by line and I've removed the line from [.config/.eslintrc](https://github.com/grafana/clock-panel/pull/179/files/39e5c0d73a36297329823703140727089ec3fd03#diff-71f3678a7bd77030e1c7ecacee33c4b70ba99e6e7f0c1dfe0243ef3e700d6e07)